docs/shortfin/llm/user/e2e_llama8b_mi300x.md

```diff
@@ -22,30 +22,37 @@ python -m venv --prompt shark-ai .venv
 source .venv/bin/activate
 ```
 
-### Install `shark-ai`
+### Install `shortfin`
 
-You can install either the `latest stable` version of `shark-ai`
-or the `nightly` version:
+You can install either the `latest stable` version of shortfin by installing
+`shark-ai` or the `nightly` version directly:
```

Comment:
You can just install `shortfin` or `shortfin[apps]` too. All the meta `shark-ai` package does is pin to matching versions of all packages.

Reply:
Yeah, I felt like it was better branding to use `shark-ai` for that part. I can switch it to `shortfin[apps]` though. What do you think?